server/superdesk/utils.py

```diff
@@ -75,6 +75,17 @@ def is_hashed(input_str):
     return input_str.startswith('$2a$')
 
 
+def merge_dicts(dict_args):
```

Comment:
That's true for our specific case, I was just thinking about a general use case. It's a minor remark anyway.